sycl/include/sycl/detail/pi.h

```diff
@@ -564,6 +566,8 @@ constexpr pi_queue_properties PI_QUEUE_OUT_OF_ORDER_EXEC_MODE_ENABLE = (1 << 0);
 constexpr pi_queue_properties PI_QUEUE_PROFILING_ENABLE = (1 << 1);
 constexpr pi_queue_properties PI_QUEUE_ON_DEVICE = (1 << 2);
 constexpr pi_queue_properties PI_QUEUE_ON_DEVICE_DEFAULT = (1 << 3);
+constexpr pi_queue_properties PI_EXT_ONEAPI_QUEUE_DISCARD_EVENTS_MODE_ENABLE =
```

Comment:
I think the mode-enable doesn't convey any additional info, let's shorten this to:
```suggestion
constexpr pi_queue_properties PI_EXT_ONEAPI_QUEUE_DISCARD_EVENTS =
```

Reply:
Fixed.